docs/quickstart.md

```diff
@@ -71,36 +70,36 @@ $ iasql new
 ├─────────────────┼───────────────────┼──────────┼──────────────────┤
 │ db.iasql.com    │ _4b2bb09a59a411e4 │ d0va6ywg │ nfdDh#EP4CyzveFr │
 └─────────────────┴───────────────────┴──────────┴──────────────────┘
-✔ As a PG connection string · postgres://d0va6ywg:nfdDh#EP4CyzveFr@db.iasql.com/_4b2bb09a59a411e4
+✔ As a PG connection string · postgres://b1qoe54n:AW1S2EW*RHdCZPkQ@127.0.0.1:5432/startup
```

Comment:
this should still use `db.iasql.com` as the host and the format of what their actual db name would be, perhaps leave the connection string as-is? it will also match the table values

Reply:
```suggestion
✔ As a PG connection string · postgres://d0va6ywg:nfdDh#EP4CyzveFr@db.iasql.com/_4b2bb09a59a411e4
```